tests/test-team/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "nhs-notify-web-template-management-ui-tests",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "test:start-local-app": "npm run dev --prefix ../../",
```

Comment:
This is a really great idea!

I wonder whether we should be running against a production build vs running in dev. OFC running `npm run dev` is faster but doing a `npm run build && npm run start` is more closer aligned with the deployed version of the app. 

There are small differences with how the app works in both modes (I remember some time ago we had really flakey PW tests due to running in dev mode). 

I'll add a comment to one of the tickets you've raised.

Reply:
updated